scripts/quests/jeuno/LB06_New_Worlds_Await.lua

```diff
@@ -0,0 +1,145 @@
+-----------------------------------
+-- New Worlds Await
+-----------------------------------
+-- Log ID: 3, Quest ID: 133
+-- Nomad Moogle : !pos 10.012 1.453 121.883 243
+-----------------------------------
+require("scripts/settings/main")
+require("scripts/globals/items")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require('scripts/globals/interaction/quest')
+local ID = require("scripts/zones/RuLude_Gardens/IDs")
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.NEW_WORLDS_AWAIT)
+-----------------------------------
+
+quest.reward =
+{
+    fame  = 50,
+    fameArea = JEUNO,
+}
+
+quest.sections =
+{
+    -- Section: Quest available. Player doesn't have Limit Breaker KI.
+    {
+        check = function(player, status)
+            return status == QUEST_AVAILABLE and
+                player:getMainLvl() >= 75 and
+                player:getLevelCap() == 75 and
+                player:hasKeyItem(xi.ki.LIMIT_BREAKER) == false and
+                xi.settings.MAX_LEVEL >= 75
+        end,
+
+        [xi.zone.RULUDE_GARDENS] =
+        {
+            ['Nomad_Moogle'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(10045, 75, 2, 10, 7, 30, 302895, 4095)
```

Comment:
Do we have any idea what these parameters do, or just from caps?

Reply:
Somewhat. event 10045 is the global event used in all Limit break quest from 6 to 10.
First argument is mostly 0 in all except here, wich happens to be either player level or level cap.
Second or third argument goes up the higher the LB quest
The rest, i have no clue. I havent tried it without those, but im inclined to believe they are just unused garbage data.

Reply:
Seems reasonable.  Can you leave a TODO with similar info there, so that if we get additional data in the future it can be reduced to whats needed?